java/simplesaga-action/src/main/java/io/simplesource/saga/action/internal/SourcingTopologyBuilder.java

```diff
@@ -0,0 +1,27 @@
+package io.simplesource.saga.action.internal;
+
+import io.simplesource.kafka.model.CommandResponse;
+import io.simplesource.saga.action.sourcing.SourcingContext;
+import io.simplesource.saga.model.specs.ActionProcessorSpec;
+import io.simplesource.saga.shared.topics.TopicConfig;
+import org.apache.kafka.streams.kstream.KStream;
+
+/**
+ * Topology builder for SourcingApp.
+ */
+public final class SourcingTopologyBuilder<A> extends AbstractActionTopologyBuilder<A> {
```

Comment:
Same as AsyncTopologyBuilder.

In this case `SourcingTopologyBuilder` has no state of its own, and uses none of the base class state, so `addSubTopology` can be made static.

Preferable would be to eliminate the class hierarchy completely.

Reply:
Gone